### PR TITLE
Fixes #2810: Added label for CompletedStoryListActivity

### LINKED ISSUE #2810: [A11y] Add label for CompletedStoryListActivity
## Task
Introduce label for `CompletedStoryListActivity` as `Stories Completed`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by ChiragKalra**
## Explanation

Fixes #2810

Added label for CompletedStoryListActivity to read "Stories Completed" when talkback is on.

Created a common string resource "completed_story_list_activity_title" for all layouts of this activty.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,7 +25,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.completedstorylist.CompletedStoryListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/completed_story_list_activity_title"/>
```
Comment (rt4914) [inline app/src/main/AndroidManifest.xml]: Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

There should be space before `/>`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ChiragKalra If you check the original issue which is #2810 there is a reference PR which contains the test case too. You need to write test case for this implementation similar to that reference PR.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ChiragKalra) [reply app/src/main/AndroidManifest.xml]: Done

File: app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt [hunk 2]
```diff
@@ -130,6 +138,23 @@ class CompletedStoryListActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  private fun createCompletedStoryListActivityIntent(): Intent {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: There is already one such function available. `createCompletedStoryListActivityIntent` use that directly.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ChiragKalra Suggested nit change.

File: app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt (hunk 2 above)
Comment (ChiragKalra) [reply app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: Done

**Comment by rt4914** [comment]
I will have to close this PR as there is **force push**. Ideally, you should not force push as it tampers the commit history.
You can create a new PR on new branch and make sure it does not contain a force push.

It is mentioned in wiki too: https://github.com/oppia/oppia-android/wiki#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me

**Pull request closed without merging**